### PR TITLE
[BD-2][BB-2365] LTI 1.3 Gating and documentation

### PR DESCRIPTION
**Pull request by giovannicimolin**
This PR gates the LTI 1.3 features, fixes a few minor bugs/rebase issues and adds documentation related to the LTI 1.3 integration.

**Testing instructions:**
1. Checkout this branch and install it on your master devstack.
2. Follow the updated README guide to set up an LTI 1.3 test tool.
3. Check that you can get a successful LTI 1.3 Launch.
4. Modify the Client ID in the LTI tool and check that an error page is displayed.
5. Check that the added documentation is ok.

**Reviewers:**
- [x] @__xitij2000__
- [ ] @davidjoy 
- [x] @davestgermain

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @giovannicimolin! I've created [OSPR-4728](https://openedx.atlassian.net/browse/OSPR-4728) to keep track of it in JIRA, where we prioritize reviews.  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

Please let us know once your PR is ready for our review and all tests are green.

File: lti_consumer/tests/unit/test_lti_1p3_consumer.py [hunk 1]
```diff
@@ -1,323 +0,0 @@
-"""
```
Comment (giovannicimolin) [inline lti_consumer/tests/unit/test_lti_1p3_consumer.py:1]: This was a result of a rebase issue, removing.

File: lti_consumer/utils.py [hunk 2]
```diff
@@ -13,6 +13,13 @@ def _(text):
     return text
 
 
+def lti_1p3_enabled():
+    """
+    Returns `true` if LTI 1.3 integration is enabled for course.
+    """
+    return settings.FEATURES.get('LTI_1P3_ENABLED', False) is True
```
Comment (xitij2000) [inline lti_consumer/utils.py:20]: I think this should be documented as well.

Comment (giovannicimolin) [reply lti_consumer/utils.py:20]: I've added a note about this in the `installation` section of the README file.

File: lti_consumer/templates/html/lti_1p3_studio.html [hunk 3]
```diff
@@ -9,6 +9,11 @@
             <b>LTI 1.3 Launches can only be performed from the LMS.</b>
         </p>
 
+        <p>
+            As of the current implementation, LTI Advantage services are not implemented, so
```
[the remark this answers is not in the document]
Comment (giovannicimolin) [reply lti_consumer/templates/html/lti_1p3_studio.html]: Done!

**Comment by giovannicimolin** [comment]
@davestgermain Sorry for the delay here. Squashed.

File: README.rst [hunk 4]
```diff
@@ -120,7 +173,7 @@ function. Below is an example:
         }
 
     dummy_processor.lti_xblock_default_params = {
-        'custom_course_id': '',
+        custom_course_id': '',
```
Comment (marcotuts) [inline README.rst:176]: is this dropped quote intentional?

Comment (giovannicimolin) [reply README.rst:176]: No, I'll put it back it. Thanks for catching that.

Comment (giovannicimolin) [reply README.rst]: @marcotuts Fixed.

**Comment by edx-webhook** [comment]
@giovannicimolin 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/xblock-lti-consumer&entry.867055334=https://github.com/edx/xblock-lti-consumer/pull/80&entry.1484655318=https://github.com/giovannicimolin&entry.752974735=2020-06-18+16:52&entry.1917517419=2020-06-25+13:03&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**